src/components/Article.jsx

```diff
@@ -1,12 +1,14 @@
 import { useEffect, useState } from "react";
-import { fetchArticleById } from "../api";
+import { fetchArticleById, fetchCommentsById } from "../api";
 import { useParams } from "react-router-dom";
 import Votes from "./Votes";
+import Comments from "./Comments";
 
 const Article = () => {
     const [article, setArticle] = useState([]);
     const [isLoading, setIsLoading] = useState(true);
     const [votes, setVotes] = useState();
+    const [comments, setComments] = useState([]);
```

Comment:
I think you could extract this out into its own component this one will become very cluttered very quickly